<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btSetup">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btHome">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>主页</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btData">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>数据</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>731</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbHeaderBottom">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(197, 229, 238);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbHeaderTop">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(89, 205, 223);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbDate">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>50</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbUser">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbTitleIcon">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>67</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbTitle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbMachineName">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbfooterbg">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>650</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 49, 49);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbLoading">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>510</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>Loading</string>
    </property>
   </widget>
   <zorder>lbfooterbg</zorder>
   <zorder>btSetup</zorder>
   <zorder>btHome</zorder>
   <zorder>btData</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>lbHeaderBottom</zorder>
   <zorder>lbHeaderTop</zorder>
   <zorder>lbDate</zorder>
   <zorder>lbUser</zorder>
   <zorder>lbTitleIcon</zorder>
   <zorder>lbTitle</zorder>
   <zorder>lbMachineName</zorder>
   <zorder>lbLoading</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
